mlos_bench/mlos_bench/optimizers/base_optimizer.py

```diff
@@ -41,7 +40,7 @@ def __init__(self, tunables: TunableGroups, service: Optional[Service], config:
         self._tunables = tunables
         self._service = service
         self._iter = 1
-        self._use_defaults: bool = bool(strtobool(str.lower(self._config.pop('use_defaults', "True"))))
+        self._use_defaults = bool(self._config.pop('use_defaults', True))
```

Comment:
Can rely on json schema validation to do the right thing here now.

Reply:
Theoretically, the user can still do `--use_defaults False` in the command line or global config and this value will be a `str` here, so we have to be careful

Reply:
Good catch, thanks!